lib/datastore/query.js

```diff
@@ -109,29 +109,27 @@ Query.prototype.autoPaginate = function(autoPaginateVal) {
  *
  * @resource [Datastore Filters]{@link https://cloud.google.com/datastore/docs/concepts/queries#Datastore_Filters}
  *
- * @param {string} filter - Property + Operator (=, <, >, <=, >=).
+ * @param {string} property - The field name.
+ * @param {string} operator - Operator (=, <, >, <=, >=).
  * @param {*} value - Value to compare property to.
  * @return {module:datastore/query}
  *
  * @example
  * // List all companies named Google that have less than 400 employees.
  * var companyQuery = query
- *   .filter('name =', 'Google')
- *   .filter('size <', 400);
+ *   .filter('name', '=', 'Google')
+ *   .filter('size', '<', 400);
  *
  * // To filter by key, use `__key__` for the property name. Filter on keys
  * // stored as properties is not currently supported.
- * var keyQuery = query.filter('__key__ =', dataset.key(['Company', 'Google']));
+ * var key = dataset.key(['Company', 'Google']);
+ * var keyQuery = query.filter('__key__', '=', key);
```

Comment:
Not 100% if this is the way to go.. since a value could potentially be anything it's hard to tell when a user is omitting an operator, is checking to see if the value is undefined a bad move?